apps/passport-client/src/dispatch.ts

```diff
@@ -1006,7 +1001,9 @@ async function syncSubscription(
     });
     onSuccess?.();
   } catch (e) {
-    onError?.(e);
+    if (e instanceof Error) {
+      onError?.(e);
+    }
```

Comment:
potentially worth creating a `toError` function whose parameter is `unknown` and always returns an `Error` for cases like this